CodingStandards.txt

```diff
@@ -91,8 +88,6 @@ All other entities' first alpha is lower case.
 4. Variable prefixes:
 
 a. Leading underscore "_" to parameter names (both normal and template).
-- Exception: "o_parameterName" when it is used exclusively for output. See 6(f).
```

Comment:
Looks like there are quite a lot of `io_` instances, actually.   Some might be I/O classes, but seeing nearly 200 instances of that string within webthree-umbrella.